src/AppBundle/Utils/Markdown.php

```diff
@@ -23,15 +23,19 @@ class Markdown
 {
     private $parser;
 
+    private $purifier;
+
     public function __construct()
     {
         $this->parser = new \Parsedown();
+        $this->purifier = new \HTMLPurifier();
```

Comment:
Could you move `HTMLPurifier` creation out to a separate class and define it as a service? I think we could create `AppBundle\Utils\Purifier` class for it and then we could inject it into the `Markdown` service. Also, in this case, we could easily use `Purifier` as a standalone service in any part of our demo application.